packages/wallets/src/features/cashier/screens/DepositLocked/DepositLocked.scss

```diff
@@ -0,0 +1,26 @@
+.wallets-deposit-locked {
+    margin-top: 9.2rem; //140px - 48px from cashier content
+
+    @include mobile {
+        margin-top: 1.6rem; //40px - 16px from cashier content
+    }
+}
+
+.wallets-deposit-locked-link {
```

Comment:
generally speaking, that should be a block within this file, to follow BEM

so something like:
.wallets-deposit-locked {
 // existing styles
 &__link {
    // styles for link
  }
}

Reply:
Ah yes! wanted to update that but forgot about it, thanks for pointing it out 😄